lib/bibliothecary/parsers/maven.rb

```diff
@@ -5,6 +5,12 @@ module Parsers
     class Maven
       include Bibliothecary::Analyser
 
+      # e.g. "annotationProcessor - Annotation processors and their dependencies for source set 'main'."
+      GRADLE_TYPE_REGEX = /^(\w+)/
+
+      # "|    \\--- com.google.guava:guava:23.5-jre (*)"
+      GRADLE_DEP_REGEX = /(\+---|\\---){1}/
```

Comment:
Need to handle the case with a resolved dep `->` see my python version

Reply:
Is that what is indicated by this? That 2.9.0 was called for and we're using 2.9.6?
"com.fasterxml.jackson.core:jackson-annotations:2.9.0 -> 2.9.6"

Reply:
that was my understanding (it was in the gradle docs someplace)